.clang-format

```diff
@@ -0,0 +1,32 @@
+---
+BasedOnStyle: Google
+AllowShortIfStatementsOnASingleLine: Never
+AllowShortLoopsOnASingleLine: false
+AllowShortFunctionsOnASingleLine: None
+ColumnLimit: 100
+IndentWidth: 4
+Language: Cpp
+TabWidth: 4
+UseTab: ForContinuationAndIndentation
+SpacesBeforeTrailingComments: 1
+BinPackArguments: false
+BinPackParameters: false
+AccessModifierOffset: -4
+SortIncludes: true
+IndentCaseLabels: true
+PointerAlignment: Left
+SortIncludes: true
```

Comment:
```suggestion
```
@aleos89 found out that this option is there twice. Gives an error during parsing of the clang format file (sometimes) in MSVS.